scripts/shared/nginx/main.conf

```diff
@@ -2,4 +2,9 @@ server {
   listen 80;
   listen [::]:80;
   include /etc/nginx/site-path-enabled/*.conf;
+  
+  root /var/www/files;
```

Comment:
I'd leave it as `root /var/www/html/` as this is the default

Reply:
Doesn't it apply to more than html though? It doesn't really matter in the end, but I feel like html would be a misnomer.